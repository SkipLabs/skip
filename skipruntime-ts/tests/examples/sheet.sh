#!/bin/bash
set -e
cd ../../examples

if [ "$#" -ne 2 ]; then
    echo "Usage: $0 <stdout target> <stderr target>"
fi

node dist/sheet.js >/dev/null &
sleep 0.25 # give a moment for service to spin up
node dist/sheet-client.js >"$1" 2>"$2"

jobs -p | xargs kill
