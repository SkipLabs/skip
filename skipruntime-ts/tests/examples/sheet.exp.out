Init []
Update [["A1",[23]],["A2",[2]]]
Update [["A3",[25]]]
Update [["A1",[5]],["A3",[7]]]
Update [["A4",[14]]]
Update [["A3",[]],["A4",["# Invalid value for cell 'A3'"]]]
